recipe/meta.yaml

```diff
@@ -16,18 +16,20 @@ build:
 requirements:
   build:
     - {{ compiler('c') }}
-    - python >=3.6
-    - make   # [not win]
+    - python
     - m2-make   # [win]
+    - make      # [not win]
     - libtool   # [not win]
   host:
-    - unixodbc
-    - postgresql
+    - unixodbc 2.3.11
+    - postgresql 17.0
   run:
     - unixodbc
-    - libpq
+    - {{ pin_compatible('libpq', max_pin='x.x.x') }}
```

Comment:
this wouldn't do anything I think because libpq isn't mentioned in the host section
it's a run_export of postgresql anyway, so doesn't need to be included

Reply:
It's needed because we build with `--with-libpq`. So technically we should also add `libpq` in the `host` deps to be explicit.

Reply:
It looks like the current version of libpq has this run_exports:
```
weak:
- libpq >=12.20,<13.0a0
```
see https://conda-metadata-app.streamlit.app/?q=pkgs%2Fmain%2Flinux-64%2Flibpq-12.20-hdbd6064_0.conda

And a new v17.0 do not change it https://github.com/AnacondaRecipes/postgresql-feedstock/pull/10/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR102

Reply:
I mean the pin_compatible doesn't do anything, sorry. adding libpq in host instead of postgresql seems to be the best option